tools/commentMonitor/client.go

```diff
@@ -95,17 +72,17 @@ func (c commentMonitorClient) verifyUser(ctx context.Context, verifyUserDisabled
 }
 
 // Extract args if regexString provided.
-func (c *commentMonitorClient) extractArgs(ctx context.Context) error {
+func (c *commentMonitorClient) extractArgs(ctx context.Context, command string) error {
```

Comment:
I know this is not relevant to this or but let add `ctx` to the `githubClient` struct and use it from the `commentMonitorClient` instead of passing here as an arg to  `extractArgs`